### PR TITLE
feat: Anima LLLite inpainting support

### PR DESCRIPTION
**Pull request by kohya-ss**
This pull request adds inpainting support to the ControlNet-LLLite training and inference scripts, enabling the use of mask images as an additional conditioning channel. The changes introduce new arguments for specifying mask images and inpainting behavior, update data pipelines to handle 4-channel (RGB+mask) conditioning, and ensure that relevant metadata is saved for reproducibility. The documentation is also updated to reflect these new features.

**Inpainting and Mask Support**

* Added functions `_load_mask_image` and `_build_inpaint_cond_image` to both `anima_minimal_inference_control_net_lllite.py` and `anima_train_control_net_lllite.py` for loading mask images and constructing 4-channel (RGB+mask) conditioning tensors. Also added batch random mask generation for training. [[1]](diffhunk://#diff-12ca0ccbe8d1c945e6a11e7130e540006564a4189853528ae1cfccd745cf7fffR77-R106) [[2]](diffhunk://#diff-232e93c0cc72afe33c04180ca7e7f04872f6b8046744a443a3eb144321d1f9e6R78-R133)
* Updated prompt parsing and argument handling to support mask images (`--mask_image`, `--mk`) and inpainting-specific options (`--lllite_cond_in_channels`, `--lllite_inpaint_masked_input`). [[1]](diffhunk://#diff-12ca0ccbe8d1c945e6a11e7130e540006564a4189853528ae1cfccd745cf7fffR194-R208) [[2]](diffhunk://#diff-12ca0ccbe8d1c945e6a11e7130e540006564a4189853528ae1cfccd745cf7fffR269-R270) [[3]](diffhunk://#diff-232e93c0cc72afe33c04180ca7e7f04872f6b8046744a443a3eb144321d1f9e6R259-R277)
* In inference and training, conditionally build 4-channel inputs and require/provide mask images when inpainting is enabled. Added warnings and error handling for missing masks. [[1]](diffhunk://#diff-12ca0ccbe8d1c945e6a11e7130e540006564a4189853528ae1cfccd745cf7fffR398-R414) [[2]](diffhunk://#diff-232e93c0cc72afe33c04180ca7e7f04872f6b8046744a443a3eb144321d1f9e6L116-R188) [[3]](diffhunk://#diff-232e93c0cc72afe33c04180ca7e7f04872f6b8046744a443a3eb144321d1f9e6R768-R777)

**Model and Metadata Handling**

* Updated model construction to pass `cond_in_channels` and `inpaint_masked_input` parameters, and to log these settings. [[1]](diffhunk://#diff-12ca0ccbe8d1c945e6a11e7130e540006564a4189853528ae1cfccd745cf7fffR324-R343) [[2]](diffhunk://#diff-12ca0ccbe8d1c945e6a11e7130e540006564a4189853528ae1cfccd745cf7fffR356-R357) [[3]](diffhunk://#diff-232e93c0cc72afe33c04180ca7e7f04872f6b8046744a443a3eb144321d1f9e6R476-R485) [[4]](diffhunk://#diff-232e93c0cc72afe33c04180ca7e7f04872f6b8046744a443a3eb144321d1f9e6R498-R499)
* Saved new metadata fields (`lllite.cond_in_channels`, `lllite.inpaint_masked_input`) in model checkpoints for reproducibility and correct inference behavior.

**Documentation**

* Extended documentation to describe the new metadata fields for inpainting support in both English and Japanese sections. [[1]](diffhunk://#diff-e8182548edbb1ed89b2cad9ab8f00daabb72f28f33a6cb24e3139cdb29c49e08R355-R356) [[2]](diffhunk://#diff-e8182548edbb1ed89b2cad9ab8f00daabb72f28f33a6cb24e3139cdb29c49e08R407-R408)

### CODE REVIEW

File: library/compile_utils.py [hunk 1]
```diff
@@ -86,6 +86,14 @@ def compile_transformer(
...
         torch._dynamo.config.cache_size_limit = args.compile_cache_size_limit
 
+    # nn.Module の tensor 属性 (例: ControlNet-LLLite が注入する self.cond_emb) は、
+    # 既定では dynamic=True でも shape が static に specialize され、解像度バケット毎に
+    # recompile を誘発する。forward 入力ではなく属性経由で渡る可変長テンソルを dynamic
+    # 対象に含めるため、この強制 static 化を無効化する。Parameter (学習重み) は形状不変
+    # なので force_parameter_static_shapes は既定 (True) のままにしておく。
+    if hasattr(torch._dynamo.config, "force_nn_module_property_static_shapes"):
+        torch._dynamo.config.force_nn_module_property_static_shapes = False
+
```
[the remark this answers is not in the document]
Comment (kohya-ss) [reply library/compile_utils.py:96]: Thanks, but this is intentional and restoring the flag here would actually break the intended behavior.

`torch.compile` is lazy: the `torch.compile(...)` calls below do not trace anything. Tracing — and the per-resolution-bucket recompiles — happen later, at forward time, throughout the whole training/inference run. So `force_nn_module_property_static_shapes` must stay `False` for the entire process lifetime, not just for the duration of this helper. Capturing and restoring the previous value right after the compile loop would re-enable static specialization before any tracing occurs, defeating the dynamic-shape handling for the `cond_emb` attribute that this block exists to enable.

This is also consistent with `cache_size_limit` a few lines above, which is likewise set as a process-wide Dynamo knob and not restored. Leaving it disabled is the design intent here.
